docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,953 @@
+
+### Organize your fleet
+
+For this tutorial, we will walk through how to set up your fleet based on the example above.
+You can choose to manage your fleet of machines differently based on what makes sense for your use case; if you're only configuring one or two sensors for personal use, feel free to add all your machines to one location and skip to the [next section](#connect-your-machines-to-the-viam-app).
+
+1. Navigate to the [Viam app](https://app.viam.com) in a web browser.
+   Create an account and log in.
+1. Click the dropdown in the upper-right corner of the **FLEET** page and use the **+** button to create a new organization for your air quality machine company.
+   Name the organization and click **Create**.
+1. Click **FLEET** in the upper-left corner of the page.
+   A new location called `First Location` is automatically generated for you.
+   Rename it so you can use it for Antonia's machines:
+
+   Click the pencil icon next to the location name, change it to `Antonia's Home`, then click **Save**.
+
+1. Now, create a separate location for RobotsRUs:
+
+   On the left side of the **FLEET** page, find the **New location** interface.
+   Type in `RobotsRUs` and click **Add**.
+
+1. Add sub-locations to the RobotsRUs location to group the machines at each of their offices:
+
+   Add a new location called `Oregon Office` using the same **New location** interface.
+   Then, find the **New parent location** dropdown on the Oregon Office page.
+   Select **RobotsRUs** and click **Change**.
+
+   Repeat to add the New York office: Add a new location called `New York Office`, then change its parent location to **RobotsRUs**.
+
+   {{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/locations-done.png" resize="x900" declaredimensions=true alt="The New York Office fleet page. The left Locations navigation panel lists Antonia's Home and RobotsRUs, with New York Office and Oregon Office nested inside RobotsRUs." >}}
+
+   In the next section, you'll add machines to the locations.
+
+## Connect your machines to the Viam app
+
+With your organizational structure in place, let's add some machines:
+
+1. Connect the single-board computer you'll be shipping to Antonia to power.
+   If the computer does not already have a Viam-compatible operating system installed, follow the [Prepare your board steps in the Installation Guide](/get-started/installation/#prepare-your-board) to install a compatible operating system.
+   You _do not_ need to follow the "Install `viam-server`" section; you will do that in the next step!
+
+1. Enable serial communication so that the SBC can communicate with the air quality sensor.
+   For example, if you are using a Raspberry Pi, SSH to it and [enable serial communication in `raspi-config`](/get-started/installation/prepare/rpi-setup/#enable-communication-protocols).
+
+1. Click **Antonia's Home** in the left navigation menu to navigate to that location's page.
```

Comment:
```suggestion
1. Add a new machine:

   Click **Antonia's Home** in the left navigation menu to navigate to that location's page.
```

Reply:
Will have a think

Reply:
let's leave as is - thanks for your patience

Reply:
Okay! :)